blank_rust_project/contract/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "status-message"
+version = "0.1.0"
+authors = ["Near Inc <hello@nearprotocol.com>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { git = "https://github.com/nearprotocol/json", rev = "1f5779f3b0bd3d2a4b0b975abc46f3d3fe873331", features = ["no_floats"] }
```

Comment:
@nearmax do we still need this given floats are allowed now?

Reply:
looks like the contract fails if we use serde_json 1.0.0 here

Reply:
Let's hold off with this change until testnet is updated. I filed an issue so that we don't forget to do this https://github.com/nearprotocol/create-near-app/issues/75